apps/learner/src/lib/components/FloatingPlayer.svelte

```diff
@@ -19,21 +15,39 @@
      * A callback function that is called when the user clicks on the play button.
      */
     onplay?: MouseEventHandler<HTMLButtonElement>;
+    /**
+     * A callback function that is called when the user clicks on the floating player.
+     */
+    onclick?: (event: MouseEvent | KeyboardEvent) => void;
```

Comment:
For non-primitive components, accepting a callback like `() => void` is preferred, because consumers typically don’t need the raw event.